docker/init.sql

```diff
@@ -0,0 +1,14 @@
+-- Create databases if they don't exist
+CREATE DATABASE keycloak;
+CREATE DATABASE shlp;
+
+-- Create users and grant privileges
+CREATE USER keycloak_user WITH PASSWORD 'keycloak_password';
+GRANT ALL PRIVILEGES ON DATABASE keycloak TO keycloak_user;
+
+CREATE USER shlp_user WITH PASSWORD 'shlp_password' CREATEDB; -- Primsa requires db create permission
```

Comment:
Note: Ive hard coded the user passwords for now just to verify this works. These ideally need to be injected as variables configured via env